base_app/post-build-step.bash

```diff
@@ -48,25 +48,35 @@ cat dist/simpleui/index.html | \
   awk -v JS_FOLDER="/${PROJ}/js/" -v CSS_FOLDER="/${PROJ}/css/" \
         '{ s=$0; \
          cssPrefix="<link rel=\"stylesheet\" href=\""; \
-         jsPrefix="<script type=\"text/javascript\" src=\""; \
+         jsPrefix="<script src=\""; \
          gsub(cssPrefix, cssPrefix CSS_FOLDER, s); \
          gsub(jsPrefix, jsPrefix JS_FOLDER, s); \
          gsub("</script>", "</script>\n", s); \
          print s;}' > dist/index.html
 
 # -- end: add GUID to js files and fix up index.html --
 
-# Copy files to /var/www
-rm -r /var/www/${PROJ}
-cp -rv dist /var/www/${PROJ}
-rm -r /var/www/${PROJ}/${PROJ}
-replace "simple_ui" "${PROJ}" -- /var/www/${PROJ}/index.html
-#ln -s /var/www/${PROJ}/sample_ui.properties.txt /var/www/${PROJ}/ui.properties
-ln -s /var/www/${PROJ}/mock-data/ui.bms.mock.properties /var/www/${PROJ}/ui.properties
+declare -A true_words_array=( [yes]=true [true]=true [1]=true [t]=true )
+if [[ -v true_words_array[${SUI_BUILD_WILL_UPDATE_LOCAL_VAR_WWW,,}] ]]; then
```

Comment:
How is this getting set? Is this an environment variable and does that mean the ansible update scripts need to incorporate it?

Reply:
The SUI_BUILD_WILL_UPDATE_LOCAL_VAR_WWW env var test was added to avoid developers inadvertently updating their build machine's `/var/www/simpleui folder` when doing `npm run build_prod` (or whatever) from console or IDE. This section would only be run when normally shouldn't be run at all.

Since I'm likely the only consumer, and not actively using the feature, I'm going to remove the entire block from the script.